### DISCUSSION TITLE
Questions about persistency stuff

### DISCUSSION THREAD

**Discussion by Manthhys** [Vdauphin/HeartsAndMinds#1385] [Question and answer] [answered]
Hi all ! Since the persistent player stuff was added to the heart and mind we're experiencing multiple problems: players being teleported mid-game while driving transport vehicles when other player connects to the last slot they used, people spawning in the middle of the forest alone... How can we circumvent the situation ? Do I have to assign a slot specific to each of my 50+ member and continuously add new slots with new recruits ? How can I assign medic/Interpreter/side mission only to certain slots ? That seems unecessarly tedious, there must be other solutions i'm not seeing.

Thanks in advance for any help !
PS: I saw the https://github.com/Vdauphin/HeartsAndMinds/discussions/1324 and https://github.com/Vdauphin/HeartsAndMinds/discussions/1363 which advices i took into account

**Comment by UnRealxInferno** [comment]
Haven't had this issue with persistence at all, as far as I know it doesn't save your current position at all

**Comment by Manthhys** [comment, marked as answer]
We found the reason for it: https://github.com/Vdauphin/HeartsAndMinds/discussions/1485#discussioncomment-5540675
